transport/transport-runtime/src/androidTest/java/com/google/android/datatransport/runtime/UploaderIntegrationTest.java

```diff
@@ -108,14 +109,12 @@ public void uploader_transientError_shouldReschedule() {
     transport.send(stringEvent);
     verify(mockBackend, times(2))
         .send(eq(BackendRequest.create(Collections.singletonList(expectedEvent))));
-    verify(spyScheduler, times(1)).schedule(any(), eq(2));
+    verify(spyScheduler, times(1)).schedule(any(), eq(2), anyBoolean());
```

Comment:
Here, we use a forced upload now. Even though this differs from the previous implementation, after reading https://github.com/firebase/firebase-android-sdk/pull/2393, I don't see why we wouldn't want to force an update - it's possible that if we hit a transient error that we still have pending events and would still want to schedule the update